<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>751</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>570</width>
        <height>81</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabBar:tab{ height: 30px; width:283px;
background-color: rgb(0,0,0);
font: 11pt; color: white;
border: 1px solid white;}
QTabBar:tab:selected  { 
    background-color : rgb(4, 98, 65) ; 
}
	</string>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>false</bool>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>지점별조회</string>
       </attribute>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>10</y>
          <width>41</width>
          <height>32</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>조회</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>8</y>
          <width>121</width>
          <height>32</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="currentText">
         <string/>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_2">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>10</y>
          <width>121</width>
          <height>32</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <item>
         <property name="text">
          <string>vip회원순</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>인기메뉴순</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="comboBox_7">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>7</y>
          <width>80</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <item>
         <property name="text">
          <string>vip회원순</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>인기메뉴순</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="comboBox_8">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>8</y>
          <width>80</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <item>
         <property name="text">
          <string>vip회원순</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>인기메뉴순</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>10</y>
          <width>80</width>
          <height>32</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>지점삭제</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>전체조회</string>
       </attribute>
       <widget class="QComboBox" name="comboBox_3">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>8</y>
          <width>160</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <item>
         <property name="text">
          <string>지점별매출순</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>인기메뉴순</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>월별매출</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>최근한달매출변화</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>10</y>
          <width>110</width>
          <height>30</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>조회</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>571</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: inset;
border-width: 1px;
border-color: gray;
background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>731</width>
        <height>192</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>320</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>지점추가</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>30</y>
        <width>151</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>rewards-logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_6">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>240</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>280</y>
        <width>131</width>
        <height>33</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="placeholderText">
       <string>지점명을 입력해주세요</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>170</y>
        <width>151</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Arial&quot;; color:white;


</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>지점관리</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_5">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>200</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Y2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>110</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_D2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>110</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Y1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_D1">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_M1">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_M2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>110</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>110</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 20pt &quot;Arial&quot;; color:gray;
</string>
      </property>
      <property name="text">
       <string>~</string>
      </property>
     </widget>
     <zorder>label_7</zorder>
     <zorder>label_5</zorder>
     <zorder>tabWidget</zorder>
     <zorder>label</zorder>
     <zorder>tableWidget</zorder>
     <zorder>pushButton_5</zorder>
     <zorder>label_3</zorder>
     <zorder>comboBox_6</zorder>
     <zorder>lineEdit</zorder>
     <zorder>comboBox_5</zorder>
     <zorder>comboBox_Y2</zorder>
     <zorder>comboBox_D2</zorder>
     <zorder>comboBox_Y1</zorder>
     <zorder>comboBox_D1</zorder>
     <zorder>comboBox_M1</zorder>
     <zorder>comboBox_M2</zorder>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLineEdit" name="lineEdit_pw">
      <property name="geometry">
       <rect>
        <x>252</x>
        <y>350</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-top: none;

border-bottom: 1px solid gray;</string>
      </property>
      <property name="text">
       <string>shs5481</string>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id">
      <property name="geometry">
       <rect>
        <x>252</x>
        <y>300</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-top: none;

border-bottom: 1px solid gray;</string>
      </property>
      <property name="text">
       <string>root</string>
      </property>
      <property name="placeholderText">
       <string>ID</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>420</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px;
background-color: rgb(255, 255, 255);
border-color: #339;
border-style: solid;
border-radius: 4;
padding: 3px;
font-size: 12px;
font: 75 9pt &quot;Arial&quot;;

</string>
      </property>
      <property name="text">
       <string>Log in</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>120</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;Arial&quot;; color:white;

</string>
      </property>
      <property name="text">
       <string>STARBUCKS</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>90</y>
        <width>171</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>rewards-logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_id</tabstop>
  <tabstop>lineEdit_pw</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>comboBox_2</tabstop>
  <tabstop>comboBox_3</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>comboBox_6</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>comboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
